homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -4,30 +4,30 @@
 from pylutron_caseta.smartbridge import Smartbridge
 import voluptuous as vol
 
+from homeassistant import config_entries
 from homeassistant.const import CONF_HOST
-from homeassistant.helpers import discovery
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 
-_LOGGER = logging.getLogger(__name__)
+from .const import CONF_CA_CERTS, CONF_CERTFILE, CONF_KEYFILE
 
-LUTRON_CASETA_SMARTBRIDGE = "lutron_smartbridge"
+_LOGGER = logging.getLogger(__name__)
 
 DOMAIN = "lutron_caseta"
-
-CONF_KEYFILE = "keyfile"
-CONF_CERTFILE = "certfile"
-CONF_CA_CERTS = "ca_certs"
+DATA_BRIDGE_CONFIG = "lutron_caseta_bridges"
 
 CONFIG_SCHEMA = vol.Schema(
     {
-        DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_HOST): cv.string,
-                vol.Required(CONF_KEYFILE): cv.string,
-                vol.Required(CONF_CERTFILE): cv.string,
-                vol.Required(CONF_CA_CERTS): cv.string,
-            }
+        DOMAIN: vol.All(
+            cv.ensure_list,
```

Comment:
I don't think we should change the config yaml schema since that is legacy now.

Reply:
I opened this PR **before** ADR-0010 was announced, and have just been making modifications as requested @MartinHjelmare 

The only change to the YAML schema is `ensure_list` so that the config accepts multiple bridges (see the PR’s original title).

This integration currently _doesn’t even use config entries_; I implemented that as part of this PR to do multiple bridges only to get this integration to be a better HA citizen and get it closer to where it can be UI-configured.

I still fully intend to make this integration actually UI configurable, but that’s not on the critical path to this PR. 

If it’s truly not mergeable because of the `ensure_list` schema change (again, this was opened before the YAML architecture decision was announced), then let me know, I guess, but I’m not sure why this PR is an issue given that it’s backwards-compatible and also gets this `lutron_caseta` integration *closer* to being YAML-less by switching to config entries.

Reply:
@MartinHjelmare so, per above, kindly let me know whether the `ensure_list` change is going to be a dealbreaker, so I know whether to fix/address what you mentioned in the other comments. Thanks.

Reply:
Ok. Keep it.